src/usr/local/www/interfaces.php

```diff
@@ -622,7 +622,7 @@ function remove_bad_chars($string) {
 			break;
 		case "dhcp6":
 			if (in_array($wancfg['ipaddrv6'], array())) {
-				$input_errors[] = sprintf(gettext("You have to reassign the interface to be able to configure as %s."), $_POST['type6']);
+				$input_errors[] = sprintf(gettext("The interface has to be reassigned to be able to configure as %s."), $_POST['type6']);
```

Comment:
Too many instances of "to"  there, and in several places in this commit, should probably be:
"The interface must be reassigned to configure as %s"